sklearn/tree/tests/test_export.py

```diff
@@ -52,6 +53,9 @@ def test_graphviz_toy():
     contents1 = export_graphviz(
```

Comment:
Can we instead make a small `for` loop? It would avoid cryptic naming of the variable.

```python
for feature_names_type in (list, np.array):
    feature_names = feature_names_type(["feature0", "feature1"])

    ...
```